### PR TITLE
allow to enable actuator per project

### PR DESCRIPTION
**Pull request by Eskibear**
Signed-off-by: Yan Zhang <yanzh@microsoft.com>

### CODE REVIEW

**Comment by Eskibear** [comment]
https://user-images.githubusercontent.com/2351748/196635701-6b595e3c-c813-4d1e-939f-25d0a579805d.mp4

File: src/models/liveProcess.ts [hunk 1]
```diff
@@ -13,9 +14,13 @@ export class LiveProcess {
         return this.liveProcess.pid;
     }
 
-    public get appName(): string {
-        const mainClass = this.liveProcess.processName;
+    public get app(): BootApp | undefined{
+        const mainClass = this.liveProcess.processName; // TODO: here assume processName is full-qualified name of mainclass
         const runningApp = appsProvider.manager.getAppByMainClass(mainClass);
-        return runningApp?.name ?? mainClass;
+        return runningApp;
```
Comment (CsCherrYY) [inline src/models/liveProcess.ts:20]: directly return `runningApp`?

Comment (Eskibear) [reply src/models/liveProcess.ts:20]: sure.

**Review by jdneo — APPROVED** [review]

**Comment by Eskibear** [comment]
found random CI failure, tracked in https://github.com/microsoft/vscode-spring-boot-dashboard/issues/230

**Pull request merged**